### PR TITLE
Small misc fixes 1

### PR DESCRIPTION
**Pull request by zeffii**
first is remove doubles. i won't push yet would like some extra eyeballs and testing..

my goal was to allow verts input without edge/poly.

just verts
image

just verts+edges
image

just verts+faces
image

not tested verts + mixed (edges/faces) yet

### CODE REVIEW

**Comment by enzyme69** [comment]
​Looks nice, a cleaner delaunay? This can work great for cloth pattern.

File: nodes/modifier_change/remove_doubles.py [hunk 1]
```diff
@@ -30,26 +30,24 @@
...
+            for edge in faces:
+                bm.edges.new([bm_verts[i] for i in edge])
+        else:
+            for face in faces:
+                bm.faces.new([bm_verts[i] for i in face])
 
     if find_doubles:
         res = bmesh.ops.find_doubles(bm, verts=bm_verts, dist=d)
-        res['targetmap']
+        res['targetmap']  # is this supposed to fail on purpose or is accidental?
```
Comment (ly29) [inline nodes/modifier_change/remove_doubles.py]: Accidental

File: nodes/modifier_change/remove_doubles.py [hunk 2]
```diff
@@ -98,15 +99,13 @@ def process(self):
         if not any([s.is_linked for s in self.outputs]):
```
Comment (ly29) [inline nodes/modifier_change/remove_doubles.py]: Could be
```python
if not any(s.is_linked for s in self.outputs):
```

File: nodes/modifier_change/remove_doubles.py [hunk 3]
```diff
@@ -98,15 +99,13 @@ def process(self):
...
+            distance = self.inputs['Distance'].sv_get(default=[self.distance])[0]
 
-            verts = Vector_generate(SvGetSocketAnyType(self, self.inputs['Vertices']))
-            polys = SvGetSocketAnyType(self, self.inputs['PolyEdge'])
-            if 'Distance' in self.inputs:
-                distance = self.inputs['Distance'].sv_get()[0]
-            else:
-                distance = [self.distance]
             if 'Doubles' in self.outputs:
                 has_double_out = bool('Doubles' in self.outputs)
```
Comment (ly29) [inline nodes/modifier_change/remove_doubles.py]: Why the ```if``` for setting a ```bool```

**Comment by zeffii** [comment]
i'll get around to this shortly.

File: nodes/modifier_change/remove_doubles.py (hunk 3 above)
Comment (zeffii) [inline nodes/modifier_change/remove_doubles.py]: i'm puzzled too :)

**Comment by zeffii** [comment]
thanks for the review, will adjust.

**Pull request merged**